datasources/fourchan/DESCRIPTION.md

```diff
@@ -1,6 +1,6 @@
 4chan data is scraped internally by 4CAT. Depending on the installation, earlier data may be imported from data dumps uploaded by the archive [4plebs](https://archive.4plebs.org) or other archives like archived.moe. Contact the administrators for details on this.
 
-<code>Deleted posts</code> are determined by whether they disappeared from the thread chain before the thread was archived. At the moment, OPs are not (accurately) checked for deletion. The replies to a deleted OP also not (yet) seen as deleted.
+<code>Deleted posts</code> are determined by whether they disappeared from the thread chain before the thread was archived. The replies to a deleted OP also seen as not deleted.
```

Comment:
"Replies to a deleted OP, however, are not marked as deleted." I think that is clearer.